tests/adjacent/adjacent.md

```diff
@@ -0,0 +1,30 @@
+# Wrens
+
+## Overview
+Wrens [^4] are a family of brown birds of which there are 88 species.
+
+## Size
+Wrens are medium-small to very small birds. The Eurasian wren is among the smallest birds in its range, while the smaller species from the Americas are among the smallest passerines in that part of the world.[^1]
```

Comment:
Hi @mjbear 
I think I don't see here an example where footnotes are placed next to each other?

e.g. I would have expected the test case to be `Wrens[^1][^4]` that is then modified to a space separated tags

Reply:
That's a reasonable expectation. I should have included one or more instances of that. Oops on my part.

All of the tags get spaced out so in the case of `world.[^1]` it would become `world. [^1]`. But that's not adjacent tags so I'll get the example fixed up with some adjacent references.

Thank you for your feedback. I don't intend to be frustrating with my mistakes.